<launch>
  <arg name="x" default="3.6" doc="X coordinate of the vehicle's initial position (in ENU)"/>
  <arg name="y" default="-2.6" doc="Y coordinate of the vehicle's initial position (in ENU)"/>
  <arg name="z" default="2.0" doc="Z coordinate of the vehicle's initial position (in ENU)"/>
  <arg name="s_2" default="false"/>
  <arg name="testing" default="false"/>
  <arg name="e_path" default="false"/>

  <!-- <node pkg="curl_navigation" type="curl_prio_image_depth.py" name="curl_node" output="screen"> -->
  <node pkg="curl_navigation" type="curl_oficial_classic.py" name="curl_node" output="screen">
    <param name="x_start"  value="$(arg x)"/>
    <param name="y_start"  value="$(arg y)"/>
    <param name="z_start"  value="$(arg z)"/>
    <param name="scenario_2"  value="$(arg s_2)"/>
    <param name="eval_path"  value="$(arg e_path)"/>
    <param name="test_param"  value="$(arg testing)"/>
    <param name="stage_number"  value="0"/>
  </node>
</launch>